src/Dumplie/Metadata/Form/TypeSchemaType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Metadata\Form;
```

Comment:
I think I would change that namespace to 'Dumplie\Metadata\Bridge\Symfony\Form\'